src/main/resources/assets/duyguji/vanilla.txt

```diff
@@ -80,3 +87,25 @@
 ☪/star-and-crescent
 ☮/peace-symbol
 ☫/farsi-symbol
+⌛/hourglass-done
+⌚/watch
+☄/comet
+✏/pencil
+✎/lower-right-pencil
+⛏/pickaxe
+⚒/hammer-and-pickaxe
+⚔/crossed-swords
+©/copyright
+®/registered-trademark
```

Comment:
I think `trademark` is intuitively enough. or maybe `trademark-registered` so it's a more obvious distinction from `tm`? Feels like I'm hardcore bikeshedding at this point =P